libraries/AP_EFI/AP_EFI_State.h

```diff
@@ -17,6 +17,16 @@
 
 #include <AP_Common/AP_Common.h>
 #include <AP_HAL/AP_HAL.h>
+#include <AP_EFI/AP_EFI_config.h>
```

Comment:
this should be moved up to just below #pragma once

... and the file content should be excluded if EFI is compiled out